models/Element/AbstractElement.php

```diff
@@ -49,7 +49,9 @@ abstract class AbstractElement extends Model\AbstractModel implements ElementInt
      */
     protected function updateModificationInfos()
     {
-        $this->setVersionCount($this->getDao()->getVersionCountForUpdate() + 1);
+        if (Model\Version::$disabled === false) {
```

Comment:
I'd prefer introducing a static getter for that. Can you change the PR accordingly? Thanks in advance!

Reply:
Hi, 
thanks for your feedback. I hope I understood you  #correctly - I added two static methods that tell you if versioning is enabled or not:

```php
# models/Version.php
/**
* @return bool
*/
public static function isDisabled(): bool
{
    return (self::$disabled === true);
}

/**
* @return bool
*/
public static function isEnabled(): bool
{
    return (self::$disabled === false);
}
```